.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+*                                    @chrisguidry @parkedwards @mitchnielsen
```

Comment:
Is there a lot of spacing between `*` and `@chris...` or is that just on my end?

Reply:
Ha, I stole this from Prefect's top-level one and it had a lot of reserved space for other paths.  Totally not necessary here, will fix